web/templates/input_pill.hbs

```diff
@@ -13,7 +13,11 @@
                 <i class="zulip-icon zulip-icon-hashtag stream-privacy-type-icon" aria-hidden="true"></i>
                 {{~/if~}}
             {{/if}}
+            {{#if is_empty_string_topic}}
+            {{sign}}topic: <span class="empty-topic-display">{{topic_display_name}}</span>
```

Comment:
Does `topic` need to be tagged for translation here? I think no, since it's an operator?